front/src/pods/student/student.component.tsx

```diff
@@ -11,25 +16,44 @@ interface Props {
 export const StudentComponent: React.FC<Props> = props => {
   const { room, log } = props;
 
+  const [autoScroll, setAutoScroll] = React.useState(false);
```

Comment:
Let's rename this to _isAutoScrollEnabled_ and _setIsAutoScrollEnabled_

```tsx
const [isAutoScrollEnabled, setAutoScrollEnabled] = React.useState(false);
```

By doing this the code is more readable, you kow exactly what does holds this state

Reply:
By default it should be true